tests/python/frontend/pytorch/test_object_detection.py

```diff
@@ -70,7 +71,7 @@ def generate_jit_model(index):
     ]
 
     model_func = model_funcs[index]
-    model = TraceWrapper(model_func(pretrained=True))
+    model = TraceWrapper(model_func(pretrained=True, rpn_pre_nms_top_n_test=200))
```

Comment:
Glad to see `rpn_pre_nms_top_n_test ` is able to limit the proposals before nms. I am not sure if the parameter is specified for real use cases, seems using default of the parameter to do benchamarking makes more sense to me

Reply:
I think the default parameter 1000 they picked is fairly conservative. This means for each level in the feature pyramid, of which there is 5 if we use resnet 50 backbones, we get maximum of 1000 x 5 boxes as input to RPN. The have another parameter `rpn_post_nms_top_n_test`, which is like topk applied after NMS. This value is also by default 1000 and it is not per class unlike `rpn_pre_nms_top_n_test`. This means we always have 1000 boxes after NMS regardless of  `rpn_pre_nms_top_n_test`.